### PR TITLE
Add range to BackupMeta; add CF

### PR DESCRIPTION
**Pull request by MyonKeminta**
Updates kvproto about raw backup & restore (https://github.com/pingcap/br/issues/86)

Adds range to `BackupMeta`.
Add cf to `BackupMeta` and `BackupRequest`.
This allows:
* Record the actual backed up range;
* Check whether the recovering range exceeds the backup data's range;
* Backup raw kv data of specified cf.

### CODE REVIEW

**Comment by MyonKeminta** [comment]
dnm. range in download request is not necessary.

**Comment by MyonKeminta** [comment]
@3pointer @overvenus PTAL

File: proto/backup.proto [hunk 1]
```diff
@@ -36,7 +36,11 @@ message BackupMeta {
     // Additional metadata describes database and table info.
     repeated Schema schemas = 7;
 
+    // If in raw kv mode, `start_versions`, `end_versions` and `schemas` will be ignored, and the
+    // backup data's range is represented by `raw_start_key` and `raw_end_key`.
     bool is_raw_kv = 8;
+    bytes raw_start_key = 9;
+    bytes raw_end_key = 10;
```
Comment (overvenus) [inline proto/backup.proto]: Could you group them into a new message?

Comment (MyonKeminta) [reply proto/backup.proto]: Done

**Pull request merged**